src/backend/joanie/settings.py

```diff
@@ -254,8 +255,10 @@ class Base(Configuration):
             "COURSE_REGEX": values.Value(
                 r"^disabled$", environ_name="MOODLE_COURSE_REGEX", environ_prefix=None
             ),
+            "COURSE_RUN_SYNC_NO_UPDATE_FIELDS": [],
```

Comment:
This seems not used by Moodle. Did we miss to implement it ?

Reply:
Yup it would be good to be able to automatically synchronize Moodle course runs on Joanie. I'll create an issue about this point.